resources/nl/cemu.po

```diff
@@ -984,929 +1181,1278 @@ msgstr ""
-
-#: MainWindow.cpp:2349
-msgid "&Dual-core recompiler (fast, unstable!)"
-msgstr "&Dual-core recompiler (snel, instabiel!)"
-
-#: MainWindow.cpp:2350
-msgid "&Triple-core recompiler (fast, unstable!)"
-msgstr "&Triple-core recompiler (snel, instabiel!)"
-
-#: MainWindow.cpp:2353
-msgid "&Mode"
-msgstr "&Modus"
-
-#: MainWindow.cpp:2354
+#: MainWindow.cpp
 msgid "&CPU"
 msgstr "&CPU"
 
-#: MainWindow.cpp:2358
+#: MainWindow.cpp
 msgid "&Scan NFC tag from file"
 msgstr "&Gebruik NFC tag van bestand"
 
-#: MainWindow.cpp:2359
+#: MainWindow.cpp
 msgid "&NFC"
 msgstr "&NFC"
 
-#: MainWindow.cpp:2363
+#: MainWindow.cpp
 msgid "&Unsupported API calls"
 msgstr "&Niet ondersteunde API calls"
 
-#: MainWindow.cpp:2364
+#: MainWindow.cpp
 msgid "&Coreinit File-Access"
 msgstr "&Coreinit File-Access"
 
-#: MainWindow.cpp:2365
+#: MainWindow.cpp
 msgid "&Coreinit Thread-Synchronization API"
 msgstr "&Coreinit Thread-Synchronization API"
 
-#: MainWindow.cpp:2366
+#: MainWindow.cpp
 msgid "&Coreinit Memory API"
 msgstr "&Coreinit Memory API"
 
-#: MainWindow.cpp:2367
+#: MainWindow.cpp
+msgid "&Coreinit MP API"
+msgstr "&Coreinit MP API"
+
+#: MainWindow.cpp
+msgid "&NN NFP"
+msgstr "&NN NFP"
+
+#: MainWindow.cpp
 msgid "&GX2 API"
 msgstr "&GX2 API"
 
-#: MainWindow.cpp:2368
+#: MainWindow.cpp
 msgid "&Audio API"
 msgstr "&Audio API"
 
-#: MainWindow.cpp:2369
+#: MainWindow.cpp
 msgid "&Input API"
 msgstr "&Input API"
 
-#: MainWindow.cpp:2370
+#: MainWindow.cpp
 msgid "&Socket API"
 msgstr "&Socket API"
 
-#: MainWindow.cpp:2371
+#: MainWindow.cpp
 msgid "&Save API"
 msgstr "&Save API"
 
-#: MainWindow.cpp:2372
+#: MainWindow.cpp
 msgid "&H264 API"
 msgstr "&H264 API"
 
-#: MainWindow.cpp:2374
+#: MainWindow.cpp
+msgid "&Graphic pack patches"
+msgstr "&Graphic pack patches"
+
+#: MainWindow.cpp
 msgid "&Texture cache warnings"
 msgstr "&Texture cache waarschuwingen"
 
-#: MainWindow.cpp:2376
+#: MainWindow.cpp
 msgid "&OpenGL debug output"
 msgstr "&OpenGL debug uitvoer"
 
-#: MainWindow.cpp:2377
-msgid "&Vulkan validation layer"
-msgstr "&Vulkan validatie laag"
+#: MainWindow.cpp
+msgid "&Vulkan validation layer (slow)"
+msgstr "&Vulkan validatie laag (traag)"
 
-#: MainWindow.cpp:2381
+#: MainWindow.cpp
+msgid "&Log PPC context for API"
+msgstr "&Log PPC context voor API"
+
+#: MainWindow.cpp
 msgid "&Textures"
 msgstr "&Textures"
 
-#: MainWindow.cpp:2382
+#: MainWindow.cpp
 msgid "&Shaders"
 msgstr "&Shaders"
 
-#: MainWindow.cpp:2383
+#: MainWindow.cpp
 msgid "&nlibcurl HTTP/HTTPS requests"
 msgstr "&nlibcurl HTTP/HTTPS verzoeken"
 
-#: MainWindow.cpp:2387
+#: MainWindow.cpp
 msgid "&Logging"
 msgstr "&Logging"
 
-#: MainWindow.cpp:2388
+#: MainWindow.cpp
 msgid "&Dump"
 msgstr "&Dump"
 
-#: MainWindow.cpp:2392 MainWindow.cpp:2397
+#: MainWindow.cpp
 msgid "&Render upside-down"
 msgstr "&Geef op zijn kop weer"
 
-#: MainWindow.cpp:2401
+#: MainWindow.cpp
+msgid "&Async compile (Vulkan)"
+msgstr "&Async compilatie (Vulkan)"
```

Comment:
I would recommend not to use the truncated form 'async' in the translation unless it's recognized as a word in your language. (I've seen that you used 'asynchrone' below)